accepted/0000-publish-prompt.md

```diff
@@ -0,0 +1,68 @@
+# Publish prompt
+
+## Summary
+
+Publishing a package should prompt a confirmation prompt, allowing users to validate their package info before uploading their package tarball.
+
+## Motivation
+
+Breaking the prompt into a two-step operation allows for a validation of contents prior to uploading the package.
+
+## Detailed Explanation and Rationale
+
+`npm publish` should ask for a confirmation prompt prior to uploading the package tarball.
+
+Currently "publish" has a very nice **unintended UX** for users with 2FA enabled, in which it stops the publish process in order to ask for the OTP code, allowing for a review of the file contents (and even cancelling the process altogether) prior to uploading the tarball file.
+
+A broader population of package authors can benefit from this UX if we formalize it. Allowing for review of contents and cancelation of package publishing.
+
+## Alternatives
+
+1. Not change it, just keep the current behavior?
+2. Implement it behind an _opt-in_ config option
+
+## Implementation
+
+This is a breaking change from the current `npm publish` behavior, it would prompt the user for confirmation:
```

Comment:
presumably in a non-TTY, it would not prompt, making it not a breaking change for automated processes, only for the human workflow?

Reply:
documented in the RFC 👍